python/README.md

```diff
@@ -34,7 +34,53 @@ def do_something(params: str):
     ...
 ```
 
-At the moment, only functions that take 0 or 1 bytes arguments will work. These functions should also either return `None` or bytes.
+At the moment, only functions that take 0 or 1 `bytes` arguments will work. These functions should also either return `None` or `bytes`.
+
+#### Resource Requirements
+
+If your function has particular resource requirements, you should specify those during registration.
+
+For example, if your function needs 4 cpu cores to run effectively, you can pass a configuration object.
+
+```python
+from preemo.worker import register
+
+@register(name="some_name", namespace="dev", cpu_cores=4)
```

Comment:
does it make more sense to have the argument be millicores so there's no question of what precision is supported

i would say you could still have a `cpu_cores` arg as an `int` for a bit of a sugar but then you have to get into function overloading or special casing around only one of the two args should be present 🤮

Reply:
it's something we can consider adding later if people run into it. I expect no one will intentionally ask for `3.0001` cores, but we'll see lol